app/models/hackbot/dispatcher.rb

```diff
@@ -2,8 +2,7 @@ module Hackbot
   class Dispatcher
     CONVERSATION_TYPES = [
       Hackbot::Conversations::CheckIn,
-      Hackbot::Conversations::Help,
-      Hackbot::Conversations::Mention
+      Hackbot::Conversations::Help
```

Comment:
Yes.

At some point over the past few weeks I think you told me I shouldn't worry so much about splitting out PRs by concern (I think this was when I was working on Hackbot more intensely). So instead of splitting the change out into a separate trivial PR, I put it here.

I'm not really a fan of it, so I'd be happy to revert this.